### PR TITLE
BaSP-MR-41: Put, Delete and Filter tests in Class

### PR DESCRIPTION
**Pull request by FrancoLelli**
With @ale-schmidt , we created tests in Class

### CODE REVIEW

**Review by lorenzomauro59 — APPROVED** [review]
It's approved for me !

**Review by TomasRomanoo — APPROVED** [review]
nice job! its approved for me!

**Review by julianv97 — APPROVED** [review]

File: src/tests/class.test.js [hunk 1]
```diff
@@ -1,5 +1,121 @@
...
+    expect(response.status).toBe(200);
+    expect(response.body.data).toBeDefined();
+    expect(response.body.error).toBeFalsy();
+    expect(response.body.message).toBe(`Class with ID ${id} updated!`);
+  });
+  test('should return status 404 when class id is invalid', async () => {
+    const invalidID = '646004aff33f9c83d28ed950';
+    const updateData = { day: 'Monday', slots: 20 };
+    const response = await request(app).put(`/api/class/${invalidID}`).send(updateData);
+    expect(response.status).toBe(500);
```
Comment (ArianaMaldonado) [inline src/tests/class.test.js:62]: The test description expects for a 404 status, but the expect is for a 500

File: src/tests/class.test.js [hunk 2]
```diff
@@ -1,5 +1,121 @@
...
+  await Class.collection.insertMany(classSeed);
+});
+afterEach(() => { jest.restoreAllMocks(); });
+
+describe('getClassById /api/class/:id', () => {
+  test('should return status 200 when id exists', async () => {
+    const id = '646004aff33f9c83d28ed954';
+    const response = await request(app).get(`/api/class/${id}`).send();
+    expect(response.status).toBe(200);
+    expect(response.body.error).toBe(false);
```
Comment (ArianaMaldonado) [inline src/tests/class.test.js:16]: We can change this line in order to keep the consistency.

```suggestion
    expect(response.body.error).toBeFalsy;
```

**Review by ArianaMaldonado — CHANGES_REQUESTED** [review]
Changes requested.

**Pull request merged**